packages/next/cms/hooks/use-comparison.js

```diff
@@ -0,0 +1,38 @@
+import {useRouter} from "next/router";
+import {useState, useEffect} from "react";
+import formatProfileResponse from "../utils/formatProfileResponse";
+import axios from "axios";
+
+
+/** */
+export default function useComparison(profile) {
+
+  const [comparison, setComparison] = useState(null);
+  const [comparisonLoading, setComparisonLoading] = useState(false);
+  const {query, locale} = useRouter();
+  const compareSlug = query?.compare;
+
+  const slug = profile.meta[0].slug;
+
+  useEffect(() => {
+    if (compareSlug) {
+      setComparisonLoading(true);
+      const url = `${process.env.NEXT_PUBLIC_CMS}profile?slug=${slug}&id=${compareSlug}&locale=${locale}`;
```

Comment:
Should this env var be declared as a `const` at the top of this file? (we probably should just test a prod build)

Reply:
if I do 
```js
const NEXT_PUBLIC_CMS = process.env.NEXT_PUBLIC_CMS;
```
it gets transpiled by tsup to:
```js
const {NEXT_PUBLIC_CMS} = process.env;
```
 and next complains. There is probably some rule that I could override but couldn't find it. Also, monshaat already works like this with no issues.